File: Wisdom-Knowledge/ariel/Ariel.java

public class Ariel {
    private String role = "Support Warlock";
    private String resource = "Mana";
    private String health = "High";
    private String damage = "None";
    private String armor = "Low";
    private String cc = "Excellent";
    private double goldShareOnKill = 0.7;

    public String q(String target) {
        if (target.equals("enemy")) return "Disorients and pushes enemy back for 2s";
        if (target.equals("ally")) return "Grants movement speed";
        return "";
    }

    public String w(String target) {
        if (target.equals("enemy")) return "Dizzies and paralyzes";
        if (target.equals("ally")) return "Heals 15% life, up to 100% gradually";
        return "";
    }

    public String e(String target) {
        if (target.equals("enemy")) return "Roots and slows";
        if (target.equals("ally")) return "Reduces skill cooldowns by 100%";
        return "";
    }

    public String r(boolean enemy, boolean ally) {
        if (enemy && ally) return "Pulls enemies to center and heals allies for 3s";
        if (enemy) return "Pulls enemies to center for 3s";
        if (ally) return "Heals allies for 3s";
        return "";
    }
}

File: Wisdom-Knowledge/ariel/ariel.py

class Ariel:
    def __init__(self):
        self.role = "Support Warlock"
        self.resource = "Mana"
        self.health = "High"
        self.damage = "None"
        self.armor = "Low"
        self.cc = "Excellent"
        self.passive = "Mana Regeneration"
        self.gold_share_on_kill = 0.7

    def q(self, target):
        if target == "enemy":
            return "Disorients and pushes enemy back for 2s"
        elif target == "ally":
            return "Grants movement speed"

    def w(self, target):
        if target == "enemy":
            return "Dizzies and paralyzes"
        elif target == "ally":
            return "Heals 15% life, up to 100% gradually"

    def e(self, target):
        if target == "enemy":
            return "Roots and slows"
        elif target == "ally":
            return "Reduces skill cooldowns by 100%"

    def r(self, targets):
        if "enemy" in targets and "ally" in targets:
            return "Pulls enemies to center and heals allies for 3s"
        elif "enemy" in targets:
            return "Pulls enemies to center for 3s"
        elif "ally" in targets:
            return "Heals allies for 3s"
